CanvasPlusPlayground/Helpers/PDFViewHelper.swift

```diff
@@ -8,21 +8,41 @@
 import SwiftUI
 import PDFKit
 
-struct BridgedPDFView: UIViewRepresentable {
+#if os(macOS)
+typealias PlatformViewRepresentable = NSViewRepresentable
+#else
+typealias PlatformViewRepresentable = UIViewRepresentable
+#endif
+
+struct BridgedPDFView: PlatformViewRepresentable {
     let pdfURL: URL
-    
+
+    #if os(macOS)
+    func makeNSView(context: Context) -> some NSView {
+        makeView(context: context)
+    }
+    #else
     func makeUIView(context: Context) -> PDFView {
+        makeView(context: context)
+    }
+    #endif
+
+    #if os(macOS)
+    func updateNSView(_ nsView: NSViewType, context: Context) {
+
+    }
+    #else
+    func updateUIView(_ uiView: PDFView, context: Context) {
+
+    }
+    #endif
+
+    func makeView(context: Context) -> PDFView {
         let pdfView = PDFView()
-        async {
+        Task {
             pdfView.document = PDFDocument(url: self.pdfURL)
         }
         pdfView.autoScales = true
         return pdfView
     }
-    
```

Comment:
Application crashes upon loading a PDF.
![CleanShot 2024-10-25 at 15 59 13@2x](https://github.com/user-attachments/assets/cef0169e-c1ba-47c8-8b1e-d11a0074be1b)

Reply:
Is the `Task` necessary here? Seems like it's attaching the document to the PDF Viewer on the BG thread after it's rendered on the main thread (via the SwiftUI view), hence the crash.

EDIT: Just tried it with `DispatchQueue.main.async` and it works fine. But the UI gets unresponsive for a small duration.

Reply:
Ok, this works without issue.
```swift
Task {
    let document = PDFDocument(url: self.pdfURL)
    DispatchQueue.main.async {
        pdfView.document = document
    }
}
```

Reply:
Fixed